nettrace/httpclient.go

```diff
@@ -225,10 +245,20 @@ func NewHTTPClient(config HTTPClientCfg, traceOpts ...TraceOpt) (*HTTPClient, er
 		netProxy:       config.Proxy,
 		pendingTraces:  lockfree.NewQueue(),
 	}
+
+	// create netTrace folder
+	if err := os.MkdirAll(netTraceFolder, 0755); err != nil {
```

Comment:
Wouldn't this fail with EEXISTS for the second call? Or is MkdirAll silent if the whole path already exists?